### ISSUE TITLE
install-maxima-jupyter sticks with lack of kernel.

### ISSUE THREAD

**Issue by robert-dodier** [robert-dodier/maxima-jupyter#58]
(I have copied the following from an email from Robert Moser.)

Dear Robert Dodier

install-maxima-jupyter sticks with lack of kernel.

What I did until now: 
1. copy of maxima-jupyter-master on my MacBook Pro (macOS High Sierra).

2. Terminal: roberts-macbook-pro:maxima-jupyter-master roebi$ python3 ./install-maxima-jupyter.py --root=`pwd`
3. Terminal: roberts-macbook-pro:maxima-jupyter-master roebi$ jupyter notebook

Jupiter launches correctly but maxima has no kernel. It seems to me that maxima uses python 2 instead of python 3. 

Here I encose the relevant part of the protocol from terminal. Now I am exhausted - I would appreciate your help. Thank you.

Best regards

Robert Moser

[20190530_2_Terminal.pdf](https://github.com/robert-dodier/maxima-jupyter/files/3237938/20190530_2_Terminal.pdf)

**Comment by yitzchak** [comment]
It's not clear to me what is happening beyond the kernel failing to execute. Maybe `maxima` is not in `PATH`? In that case specifying `--exec` during install would help. Also, I would try using `jupyter console --kernel=maxima` instead of the notebook as the console messages will probably be more informative.

**Comment by roebix** [comment]
[summary.nb.pdf](https://github.com/robert-dodier/maxima-jupyter/files/3296619/summary.nb.pdf)

Thank you for help
Robert

**Comment by yitzchak** [comment]
Please try using the `--exec` option to explicitly specify the path to your maxima binary. If you only have the SageMath maxima binary then this would be:

```
python3 ./install-maxima-jupyter.py --root=`pwd` --exec=/Applications/Maxima.app/Contents/Resources/opt/bin/maxima
```

**Comment by robert-dodier** [comment]
If I am not mistaken, Sage builds Maxima with ECL. I don't know if Maxima-Jupyter can work with ECL.

**Comment by roebix** [comment]
After a third trial the installation succeded. But now there is another problem (see file Connection failed.png).

[20190619.nb.pdf](https://github.com/robert-dodier/maxima-jupyter/files/3307477/20190619.nb.pdf)

![Connection failed](https://user-images.githubusercontent.com/28433982/59791070-9826a380-92d1-11e9-8b64-046ea725d3f5.png)

Robert

**Comment by yitzchak** [comment]
@robert-dodier You are correct, the SageMath binary won't work. I forgot to update my intro text but used the path to the correct binary in my example.

@roebix Please try `jupyter-console --kernel=maxima` as this will more info to the console.

**Comment by roebix** [comment]
It is fairly confusing task!

[20190620.pdf](https://github.com/robert-dodier/maxima-jupyter/files/3311252/20190620.pdf)

**Comment by yitzchak** [comment]
It looks like the maxima script cannot find SBCL. You should probably make sure the sbcl binary is in `PATH`.

**Comment by roebix** [comment]
Unfortunately it doesn't work with SBCL either.

[apple, how extend $PATH in high sierra.pdf](https://github.com/robert-dodier/maxima-jupyter/files/3316720/apple.how.extend.PATH.in.high.sierra.pdf)
![20190622](https://user-images.githubusercontent.com/28433982/59962566-89193e80-94e7-11e9-8d2e-7695b9c95641.png)

**Comment by yitzchak** [comment]
Your export command probably should have been

```sh
export PATH=”$PATH:/Applications/Maxima.app/Contents/Resources/opt/bin”
```

**Comment by roebix** [comment]
Unfortunately, the same problem is again apparent, there is no kernel.
Now I give up. Thanks for the friendly help. Best regards Robert Moser
